sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -1074,11 +1070,18 @@ object DDLUtils extends Logging {
       query: LogicalPlan,
       outputPath: Path,
       table: Option[CatalogTable] = None) : Unit = {
-    val inputPaths = query.collect {
-      case LogicalRelationWithTable(r: HadoopFsRelation, _) =>
+    def getPaths(query: LogicalPlan): Seq[Path] = query.collect {
+      case LogicalRelation(r: HadoopFsRelation, _, _, _) =>
```

Comment:
Why do I see this change (replacing LogicalRelationWithTable by LogicalRelation) in this CR? This change is already present in the latest code.

Reply:
> Why do I see this change (replacing LogicalRelationWithTable by LogicalRelation) in this CR? This change is already present in the latest code.

@zeal-thinker It looks like something has changed in this class since I submitted this PR, causing a code conflict. So, I created a new PR that inherits from the latest code. Please take a moment to review the following link? https://github.com/apache/spark/pull/48640.